scripts/sysman/powerLimits.yml

```diff
@@ -86,6 +91,7 @@ details:
     - "The application may call this function from simultaneous threads."
     - "The implementation of this function should be lock-free."
     - "This function returns all the power limits associated with the supplied power domain."
+    - "1.7": "Note: This function is deprecated and functionality is provided by $sPowerGetPowerLimits."
```

Comment:
`- "1.7": "[DEPRECATED] Use $sPowerGetPowerLimits."`

Reply:
need to define new set of interfaces first